src/components/tags/tags.js

```diff
@@ -111,8 +127,39 @@ export class TagsComponent {
     );
   }
 
-  handleDeleteClick() {
-    console.log("handleDeleteClick");
+  handleDeleteClick(event) {
+    let moreButton = event.target.closest(".button-more");
+    this.listItem = moreButton.closest(".tags__list-item");
+    let tagName = this.listItem.querySelector(".tags__list-item-name")
+      .innerText;
+    this.confirmDialog.showDialog("tag", tagName);
+  }
+
+  initConfirmDialog() {
+    this.confirmDialog = new ConfirmDialogComponent(
+      this.confirmDialogMountPoint,
+      {
+        onOkClick: this.handleDeleteConfirm.bind(this)
+      }
+    );
+    this.confirmDialog.mount();
+  }
+
+  handleDeleteConfirm() {
+    this.delTag(this.listItem);
+  }
+
+  delTag(listItem) {
+    let tag = listItem.querySelector(".tags__list-item-name").innerText;
+    TransactionListService.deleteByTag(tag)
+      .then(() => {
```

Comment:
`TransactionListService.deleteByTag(tag)`
`.then(() => TagListService.del(tag)})`
`.then(() => {
       this.tagsList.removeChild(this.tagToDel);
        this.props.onTagDelete();
   });`